src/Goteo/Model/Faq/FaqSection.php

```diff
@@ -111,63 +98,82 @@ static public function getList(array $filters = [], int $offset = 0, int $limit
               $sql
               ORDER BY faq_section.order ASC
               LIMIT $offset, $limit";
-         //die(\sqldbg($sql, $values));
         $query = static::query($sql, $values);
         return $query->fetchAll(\PDO::FETCH_CLASS, __CLASS__);
     }
 
+     static public function getListCount(array $filters = [], $lang = null): int
+     {
+         if(!$lang) $lang = Lang::current();
+         list($fields, $joins) = self::getLangsSQLJoins($lang, Config::get('sql_lang'));
+
+         $filter = [];
+         $values = [];
+
+         if($filter) {
+             $sql = " WHERE " . implode(' AND ', $filter);
+         }
+
+         $sql = "SELECT
+              count(faq_section.id)
+          FROM faq_section
+          $joins
+          $sql";
+         $query = static::query($sql, $values);
+         return $query->fetchColumn();
+    }
 
-  public function getFaqbySection(int $limit = 0): array
-  {
-
-    return Faq::getList(['section'=>$this->id], 0, $limit);
-
-  }
+    /**
+     * @return Faq[]
+     */
+    public function getFaqs(int $limit = 0): array
+    {
+        $count = Faq::getListCount(['section' => $this->id]);
+        return Faq::getList(['section'=>$this->id], 0, $count);
+    }
```

Comment:
@davidbeig The parameter `$limit` is not being used inside the method. Maybe we could just remove it?